tests/acceptance/features/api/api_sync_region.feature

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+Feature: Synchronize the images of the corresponding region defined by its regionId.
+         The operation is asynchronous it responses with a taskId in order that
+         you can follow the execution of the process.
+
+  As a GlanceSync API user
+  I want to sync images in a region using the GlanceSync API
+  So that I can have all FIWARE images up to date in all FIWARE nodes.
+
+  GlanceSync API operation under testing:
+    POST regions/regionId
+
+
+  @happy_path @env_dependant @experimentation @skip # needs a real environment.
```

Comment:
I would need to undestarnd the different tags. I know skip but what are the others.